site/blog.html

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Blog / News
+permalink: /blog/
+---
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+
+<h2>Project News and Blog</h2>
+<hr></hr>
```

Comment:
`<hr></hr>` -> `<hr>`

http://w3c.github.io/html-reference/syntax.html#void-element

> Void elements only have a start tag; end tags must not be specified for void elements.

We can't specify `<hr/>`.

> Optionally, a "/" character, which may be present only if the element is a void element.

We can write `<hr>` or `<hr/>`.